Kontent.Ai.Management.Tests/ManagementClientTests/ProjectUserTests.cs

```diff
@@ -3,76 +3,155 @@
 {
-    private readonly FileSystemFixture _fileSystemFixture;
+    private readonly Scenario _scenario;
 
-    public ProjectUserTests(FileSystemFixture fileSystemFixture)
+    public ProjectUserTests()
     {
-        _fileSystemFixture = fileSystemFixture;
-        _fileSystemFixture.SetSubFolder("ProjectUser");
+        _scenario = new Scenario(folder: "ProjectUser");
     }
 
     [Fact]
-    public async Task InviteUser_InvitesUser()
+    public async Task InviteUserIntoProjectAsync_InvitesUser()
     {
-        var client = _fileSystemFixture.CreateMockClientWithResponse("ProjectUser.json");
-
-        var expected = _fileSystemFixture.GetExpectedResponse<UserModel>("ProjectUser.json");
+        var client = _scenario
+            .WithResponses("ProjectUser.json")
+            .CreateManagementClient();
 
         var invitation = new UserInviteModel
         {
             Email = "test@kontent.ai",
-            CollectionGroup = expected.CollectionGroup
+            CollectionGroup = new[] { 
+                new UserCollectionGroup
+                {
+                    Collections = new [] { Reference.ById(Guid.NewGuid()), Reference.ById(Guid.NewGuid()) },
+                    Roles = new[] {
+                        new RoleModel
+                        {
+                            Id = Guid.NewGuid(),
+                            Languages = new [] { Reference.ById(Guid.NewGuid()), Reference.ById(Guid.NewGuid()) }
+                        }
+                    }
+                }
+            }
         };
 
         var response = await client.InviteUserIntoProjectAsync(invitation);
 
-        response.Should().BeEquivalentTo(expected);
+        _scenario
+            .CreateExpectations()
+            .HttpMethod(HttpMethod.Post)
+            .RequestPayload(invitation)
+            .Response(response)
+            .Url($"{Endpoint}/projects/{PROJECT_ID}/users")
+            .Validate();
     }
 
+    
     [Fact]
-    public async Task InviteUser_UserInvitationModelNotProvided_ThrowsException()
+    public async Task InviteUserIntoProjectAsync_UserInvitationModelIsNull_Throws()
     {
-        var client = _fileSystemFixture.CreateMockClientWithResponse("ProjectUser.json");
+        var client = _scenario.CreateManagementClient();
 
         await client.Invoking(x => x.InviteUserIntoProjectAsync(null)).Should().ThrowExactlyAsync<ArgumentNullException>();
     }
 
+    
     [Fact]
-    public async Task ModifyUsersRole_ByEmail_ModifiesUserRoles()
+    public async Task ModifyUsersRolesAsync_ByEmail_ModifiesUserRoles()
     {
-        var client = _fileSystemFixture.CreateMockClientWithResponse("ProjectUser.json");
+        var client = _scenario
+            .WithResponses("ProjectUser.json")
+            .CreateManagementClient();
 
-        var expected = _fileSystemFixture.GetExpectedResponse<UserModel>("ProjectUser.json");
+        var request = new UserModel
```

Comment:
I think we should rename the `request` variable to some better name. Same for _ModifyUsersRolesAsync_ById_ModifiesUserRoles_